crates/whiskers/src/grid/mod.rs

```diff
@@ -0,0 +1,181 @@
+//!
+use vsvg::{Draw, Point};
+
+use crate::Sketch;
+
+use self::cell::GridCell;
+
+pub mod cell;
+
+/// Grid's size can be set either by passing
+/// the cell's or grid's dimensions. Pass one of the enum members
+/// to choose
+pub enum GridSize {
+    /// Set cell size, grid's size will be computed
+    CellBased([f64; 2]),
+    /// Set fixed grid size
+    GridBased([f64; 2]),
+}
```

Comment:
Ideally we can make this sort of thing an implementation detail (i.e. not `pub`)  and expose it through nice builder pattern.